lib/views/message.coffee

```diff
@@ -1,14 +1,14 @@
 # This file is imported in views/panel
 
 class Message extends HTMLElement
-  initialize: (@message, @addPath) ->
+  initialize: (@message, @options) ->
 
   attachedCallback: ->
     @appendChild Message.renderRibbon(@message)
-    @appendChild Message.renderMessage(@message)
-    @appendChild Message.renderLink(@message, @addPath) if @message.filePath
+    @appendChild Message.renderMessage(@message, @options)
+    @appendChild Message.renderLink(@message, @options) if @message.filePath
 
-  @renderLink: (message, addPath) ->
+  @renderLink: (message, {addPath}) ->
```

Comment:
Bad things will happen yes, but atleast they'll know that they did something they shouldn't have.